Source/ResponseSerialization.swift

```diff
@@ -24,11 +24,9 @@
 
 import Foundation
 
-// MARK: ResponseSerializer
-
-/// The type in which all response serializers must conform to in order to serialize a response.
-public protocol ResponseSerializerType {
-    /// The type of serialized object to be created by this `ResponseSerializerType`.
+/// The type in which all data response serializers must conform to in order to serialize a response.
+public protocol DataResponseSerializerType {
```

Comment:
I believe the Swift 3 guidelines call for these sorts of protocols to end in `Protocol` now, not `Type`.